pkg/provider/azure_loadbalancer.go

```diff
@@ -1660,7 +1660,7 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 			for i := range toDeleteConfigs {
 				fipConfigToDel := toDeleteConfigs[i]
 				err := az.reconcilePrivateLinkService(clusterName, service, &fipConfigToDel, false /* wantPLS */)
-				if err != nil {
+				if err != nil && !retry.HasStatusForbiddenOrIgnoredError(err) {
```

Comment:
Could we skip 403/404 only for cleanup operation? For create operation, 403/404 should not be skipped.

Reply:
Thanks for reminding me.
Compared to last commit, I use `HasStatusForbiddenOrIgnoredError` only for PLS and backend pool because I think the issue is about these resources instead of those LB's resources like FIP.
Code updated.